iothub_client/inc/internal/iothubtransport_amqp_device.h

```diff
@@ -106,23 +106,23 @@ typedef struct DEVICE_CONFIG_TAG
 
```

Comment:
I would suggest renaming DEVICE_CONFIG to IOTHUBTRANSPORT_AMQP_DEVICE_CONFIG. Same for the _TAG above (rename DEVICE_CONFIG_TAG to IOTHUBTRANSPORT_AMQP_DEVICE_CONFIG_TAG).
Everything else is good. Thanks for this.